packages/jest-message-util/package.json

```diff
@@ -8,6 +8,7 @@
   "license": "MIT",
   "main": "build/index.js",
   "dependencies": {
+    "@babel/code-frame": "^7.0.0-beta.35",
```

Comment:
How much stuff does this pull into Jest?

Reply:
Check the lockfile.

```
"@babel/code-frame@^7.0.0-beta.35":
  version "7.0.0-beta.35"
  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0-beta.35.tgz#04eeb6dca7efef8f65776a4c214157303b85ad51"
  dependencies:
    chalk "^2.0.0"
    esutils "^2.0.2"
    js-tokens "^3.0.0"
```

We can go with the babel 6 version which we already have as it's used by babel itself when transpiling, and just use this one whenever we bump to babel-core@7 internally

Reply:
Oh yeah, good point. This is fine then.